include/swift/AST/DiagnosticsSema.def

```diff
@@ -3985,6 +3985,11 @@ ERROR(dynamic_with_transparent,none,
       "a declaration cannot be both '@_tranparent' and 'dynamic'",
       ())
 
+ERROR(dynamic_and_library_evolution_not_supported,none,
+      "a native Swift declaration cannot be marked 'dynamic' in library evolution mode",
```

Comment:
Can you change this to `non-@objc declarations` instead of "a native Swift declaration"? We don't use the term "native" in diagnostics.

Reply:
Sure.